# HG changeset patch
# User Torsten <mttl@mailbox.org>
# Date 1512301033 -3600
#      Sun Dec 03 12:37:13 2017 +0100
# Node ID 222addbadf46f46805ea505171e2ea86243beec2
# Parent  f77da8da0f3f0b331414906af7c94b5990cb690b
fix truncated startup message in the gui terminal

* main-window.cc (set_window_layout): only restore the main window layout
  and state before all dock widgets geometry are restored

diff --git a/libgui/src/main-window.cc b/libgui/src/main-window.cc
--- a/libgui/src/main-window.cc
+++ b/libgui/src/main-window.cc
@@ -1300,19 +1300,16 @@ main_window::set_window_layout (QSetting
                 widget->make_widget (false);  // no docking, just reparent
 
               widget->make_widget ();
               widget->setVisible (visible);   // not floating -> show
             }
         }
     }
 
-  restoreState (settings->value ("MainWindow/windowState").toByteArray ());
-  restoreGeometry (settings->value ("MainWindow/geometry").toByteArray ());
-
   show ();
 }
 
 void
 main_window::write_settings (void)
 {
   QSettings *settings = resource_manager::get_settings ();
   if (! settings)
